ansible/roles/test/files/ptftests/l2_lag.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import random
+
+import ptf
+import ptf.dataplane as dataplane
```

Comment:
Consider using [ptfadapter](https://github.com/Azure/sonic-mgmt/blob/master/tests/ptfadapter/README.md) instead of writing a separate ptftest? 
It helps verify traffic from the test file itself, no need to copy it over to ptf docker. See [FDB](https://github.com/Azure/sonic-mgmt/blob/master/tests/fdb/test_fdb.py) for examples.

Reply:
ptfadapter link: https://github.com/Azure/sonic-mgmt/tree/b6a5f28e587c878f6289b8f333dd4e3ba1a8ddbd/tests/common/plugins/ptfadapter

Reply:
@mykolaf @yxieca 
Using ptfadapter instead of writing a separate ptftest.